runtime/onert/core/src/exec/MultiModelExecutors.cc

```diff
@@ -486,18 +476,14 @@ void MultiModelExecutors::execute(const ExecutionContext &ctx)
       else
       {
         auto from_iodesc = find_from(model_index, ir::SubgraphIndex{0}, ir::IOIndex{i});
-        const auto &from_model_index = std::get<ir::ModelIndex>(from_iodesc);
-        const auto &from_subg_index = std::get<ir::SubgraphIndex>(from_iodesc);
-        const auto &from_ioindex = std::get<ir::IOIndex>(from_iodesc).value();
 
         // Supported only sequantial execution of models
-        assert(from_model_index.value() < model_index.value());
-        assert(from_subg_index.value() == 0);
-        const auto from_executor = _executors.at({from_model_index, from_subg_index}).get();
+        assert(std::get<ir::ModelIndex>(from_iodesc).value() < model_index.value());
+        assert(std::get<ir::SubgraphIndex>(from_iodesc).value() == 0);
         const auto to_iodesc = ir::IODesc{model_index, ir::SubgraphIndex{0}, ir::IOIndex{i}};
         if (_edge_quant_tensors.find(to_iodesc) == _edge_quant_tensors.end())
         {
-          inputs_inter[i] = from_executor->getOutputTensors().at(from_ioindex);
+          inputs_inter[i] = _edge_tensors.at(from_iodesc).get();
```

Comment:
`from_executor->getOutputTensors().at(from_ioindex)` points same tensor in `_edge_tensors.at(from_iodesc).get()`